library/src/main/java/com/google/android/material/motion/runtime/Scheduler.java

```diff
@@ -147,7 +147,11 @@ public void removeStateListener(StateListener listener) {
    * Commits the given {@link Transaction}. Each {@link PlanInfo} is committed in the context of
    * its target, called a {@link TargetScope}. Each TargetScope ensures that only one instance of a
    * specific type of Performer is created.
+   * @deprecated  Post version 1.0, plans should be added directly to the Scheduler instead of using Transactions. </br>
+   *              This will be removed in the next version </br>
+   *              use {@link #addPlan()} on the Scheduler instead
    */
+  @Deprecated
```

Comment:
Sure - should be taken care of in https://github.com/material-motion/material-motion-runtime-android/pull/36/commits/37dac4452fdb77b0883108315fa479f94e1a8d37. Haven't deprecated the `addNamedPlan` in the `Transaction` as I think it's probably best to do that in https://github.com/material-motion/material-motion-runtime-android/issues/34?